api/SpeechGrammarList.json

```diff
@@ -20,7 +20,14 @@
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
-            "version_added": false
+            "version_added": "142",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "media.webspeech.recognition.enable",
+                "value_to_set": "true"
+              }
+            ]
```

Comment:
FWIW, Firefox also supports the `webkit` prefix for these interfaces and their constructors:

<img width="241" height="119" alt="image" src="https://github.com/user-attachments/assets/691ecae4-6135-49c6-8c66-87ef98bb0e9f" />

Reply:
OK, done